### ISSUE TITLE
Getting "failed to read contents" after line 5

### ISSUE THREAD

**Issue by Casuallynoted** [Cloudef/guhck#1]
Hello!

I was able to build Guhck, but when I run it on any of the CCS files I've extracted, it ultimate ends with something like this (and doesn't export any models):

(0, 16) 857805056
failed to read contents

Is there anything I can do to get this working?
Thanks!

**Comment by Cloudef** [comment]
The ccs files are gzip compressed, decompress them first.

**Comment by Cloudef** [comment]
Actually never mind, it seems I've forgot that this tool already decompresses. Maybe the file just won't be read with guhck? Not everything works there.

**Comment by Casuallynoted** [comment]
Hmm.... what's weird is it seems to be doing this with just about every CCS file I run through it. Would you mind if I sent you the CCS file? :o

**Comment by Cloudef** [comment]
There's lots of models (characters for example) that won't work. Images and simple models like the bikes should work.

**Comment by Casuallynoted** [comment]
@Cloudef Ahh yeah, I just noticed that the bikes do extract, and that the character files show animation files in the list but don't extract anything. Is there any way to get even just the area models working by any chance? :o I've been able to rip most of each area using CCS2OBJ but it doesn't rip all of it. I'd be willing to support development of a fully featured CCS exporting tool.

**Comment by Cloudef** [comment]
Maybe the source albeit ugly can work as reference. I don't currently have any plans to continue it. Some communities such as Xentax may be able to help you figuring out everything from CCS files.

**Comment by Casuallynoted** [comment]
Absolutely, I think I'll probably get in touch with some people over there who might be willing to help me work this out. :) Thank you so much!
